Dungeon_of_Munchkinomicon.py

```diff
@@ -104,14 +112,14 @@ def room1(name):
 		print "Invalid action"
 		room1(name)
 
-def reddoor(name):
+def redDoor(name):
 	global fairy
 	global katana
 	print line
 	print "You are now in what looks like a broom closet."
 	print "What do you do?"
-	redaction = raw_input(prompt)
-	if "search" in redaction or "loot" in redaction and not fairy:
+	action = raw_input(prompt)

```

Comment:
At the beginning of red_door, fairy_chamber, and cuthulu_chamber you allow the players an action. So that new contributors see the similarities between the variables I renamed them all to action.